lib/pbench/test/unit/server/conftest.py

```diff
@@ -184,7 +184,7 @@ def login_user(client, server_config, username, password):
 
 
 @pytest.fixture()
-def create_user() -> User:
+def create_user(client) -> User:
```

Comment:
It uses the DB; in previous contexts the test case already included the `client` fixture directly, but I thought it was more appropriate to reference it here rather than add it to my new exception test class where it's not used directly.

Reply:
OK.  Perhaps a comment would be helpful.